demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -153,6 +159,12 @@ def prepared_graph_repo(graph_repo: Repo):
     sample_pack_3.set_data(marketplaces=MP_XSOAR)
     sample_pack_3.create_script("SampleScriptTwo").set_data(marketplaces=MP_XSOAR)
 
+    sample_pack_4 = graph_repo.create_pack("SamplePack4")
+    sample_pack_4.set_data(marketplaces=MP_XSOAR_AND_V2)
+    sample_pack_4.create_integration(name="SampleIntegration")
+    # duplicate integration as in sample_pack for testing GR 105
```

Comment:
this is an assumption that may not always be true (or, can be broken easily) - add an `assert` above it to make sure such an integration exists